doc/prop.xml

```diff
@@ -788,3 +788,38 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="IsHamiltonianDigraph">
+<ManSection>
+  <Prop Name="IsHamiltonianDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is Hamiltonian, this property returns
+    <K>true</K>.<P/>
+
+    A Hamiltonian digraph is a digraph which has a circuit visiting 
+    each vertex exactly once.
+    Note the empty digraphs on 0 and 1 vertices are considered 
```

Comment:
Do you mean digraphs with no edges? Or do you just mean "any digraph with 0 or 1 vertices is Hamiltonian."? I note in the code you really be a digraphs with 0 or 1 vertices and no edges, so a digraph with an 1 vertex and a loop is not Hamiltonian?